src/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Steve's To-Do-List</title>
+    </head>
+    <body>
+        <section class="d-flex-column">
+            <div>
+                <form action="#" class="to-do-add-form">
+                    <div class="d-flex-align-center heading-wrapper row">
+                        <h1 class="to-do-heading no-spacing">Today's To Do</h1>
+                        <button type="button" class="to-do-refresh-btn">
+                            <i class="fa-solid fa-rotate"></i>
+                        </button>
+                    </div>
+                    <div class="d-flex-align-center field-wrap row">
+                        <input type="text" name="desc" placeholder="Add to your list.." required>
+                        <span><i class="fa fa-level-down"></i></span>
+                    </div>
+                </form>
+                <ul class="d-flex-column to-do-list no-spacing"></ul>
+                <button type="button" class="to-do-clear-btn" disabled>Clear all completed</button>
```

Comment:
Your class naming convention looks a bit long. Kindly try to use short and precise naming conventions